tests/core/config/bdk_config_test.py

```diff
@@ -0,0 +1,32 @@
+from symphony.bdk.core.config.loader import BdkConfigLoader
+
+from tests.utils.resource_utils import get_config_resource_filepath
+import pytest
+
```

Comment:
I was missing inspiration for unit test as at the end I'm testing setters..

Reply:
The configuration should be either RSA or certificate exclusively (we cannot have both). And you should also test that BotAuthenticatorRsa uses the private key content when specified while authenticating